amazon-kinesis-client/src/main/java/software/amazon/kinesis/coordinator/CoordinatorConfig.java

```diff
@@ -90,4 +90,10 @@
 
     private CoordinatorFactory coordinatorFactory = new SchedulerCoordinatorFactory();
 
+    /**
+     * The number of milliseconds to sleep in between logging the executor state.
+     *
+     * <p>Default value: 30,000 milliseconds</p>
+     */
+    private long executorDiagnosticsDaemonSleepTimeMillis = 30000L;
```

Comment:
Not needed: Seems to be a long name, if it can be made shorter would be better.

Reply:
Renamed